python/pyspark/sql/tests.py

```diff
@@ -4090,6 +4090,25 @@ def __init__(self, **kwargs):
                 _make_type_verifier(data_type, nullable=False)(obj)
 
 
+class HaveArrowTests(unittest.TestCase):
+
+    @unittest.skipIf(_have_pandas and _have_pyarrow,
+                     "Required PyArrow and Pandas were found, Arrow tests will run")
+    def test_required_pyarrow_pandas_not_installed(self):
```

Comment:
@HyukjinKwon any idea why this wouldn't have printed a skip message on the last 3 out of 5 tests?

Reply:
So looking at https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/95514/consoleFull it's printing the other skip method ( "Required PyArrow and Pandas not found") so maybe only some of the Jenkins workers have the correct version of PyArrow and Pandas installed? We should probably check that?